src/main/webapp/i18n/de/textExercise.json

```diff
@@ -14,6 +14,8 @@
             },
             "wordCount": "Anzahl der Wörter: {{ count }}",
             "characterCount": "Anzahl der Zeichen: {{ count }}",
+            "create": "Text erstellen",
+            "import": "Text importieren",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys